app/models/restaurant.rb

```diff
@@ -8,4 +8,20 @@ class Restaurant < ActiveRecord::Base
   validates :city, presence: true
   validates :state, presence: true
   validates :zip_code, presence: true
+
+  def average_rating
+    self.reviews.sum(:rating)/self.reviews.count
```

Comment:
I played around with this, it doesn't perform properly without being declared on 'self', not 100% sure why though : )